.github/workflows/docker push-package-base.yml

```
@@ -9,18 +9,24 @@ jobs:
   package-base:
     runs-on: ubuntu-20.04
     steps:
-    
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
+
     - name: Set up Docker Buildx
-      uses: docker/setup-buildx-action@v1
+      uses: docker/setup-buildx-action@v2
 
     - name: Login to DockerHub
-      uses: docker/login-action@v1
+      uses: docker/login-action@v2
       with:
         username: ${{ secrets.DOCKERHUB_USERNAME }}
         password: ${{ secrets.DOCKERHUB_TOKEN }}
 
     - name: Build and push
       uses: docker/build-push-action@v2
       with:
+        platforms: linux/amd64,linux/arm64
         push: true
-        tags: openhie/package-base:${{github.ref_name}}
+        tags: tryzan/package-base:${{github.ref_name}}
```

```suggestion
        tags: openhie/package-base:${{github.ref_name}}
```
This should stay the same